### PR TITLE
Feature/loginpage

### PR DESCRIPTION
**Pull request by RobertoWTBW**
Login page was implemented as seen by the design. A bug is also present => textFiele outllineborder does not become red

### CODE REVIEW

**Comment by CimpeanAndreea** [comment]
Please rename the first 2 files in the assets folder to be more suggestive

File: lib/constants/constants.dart [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const titleFontSize = 38.0;
+const subtitleFontSize = 24.0;
+const secondSubtitleFontSize = 18.0;
+const logoBlue = 0xff264653;
+const coolGreen = 0xff2a9d8f;
```
Comment (CimpeanAndreea) [inline lib/constants/constants.dart]: What is this? Can it be named more suggestive?

File: lib/loginpage/loginpage.dart [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  Widget build(BuildContext context) {
+    final screenWidth = MediaQuery.of(context).size.width;
+    final screenHeight = MediaQuery.of(context).size.height;
+    return Scaffold(
+        body: Container(
+            height: double.infinity,
+            width: double.infinity,
+            decoration: const BoxDecoration(
+                image: DecorationImage(
+                    image: AssetImage("assets/gradient.png"),
```
Comment (CimpeanAndreea) [inline lib/loginpage/loginpage.dart]: Please create a file for constants to define these paths

Comment (RobertoWTBW) [reply lib/loginpage/loginpage.dart]: Done

File: lib/constants/constants.dart (hunk 1 above)
Comment (RobertoWTBW) [reply lib/constants/constants.dart]: Done

**Pull request merged**